fotoobo/fortinet/forticloud.py

```diff
@@ -0,0 +1,194 @@
+"""
+FortiCloud Class
+"""
+
+import logging
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+import requests
+
+from fotoobo.exceptions.exceptions import APIError, GeneralWarning
+
+from .fortinet import Fortinet
+
+log = logging.getLogger("fotoobo")
+
+
+class FortiCloudAsset(Fortinet):
```

Comment:
Is there a reason why it is called "FortiCloudAsset" here but everywhere else is just "forticloud" or "fc"? I would either rename the file to "forticloud_asset" or "fca" or the class just to "FortiCloud" to approximately keep with the baseline 1 file = 1 class.